packages/sui-ssr/server/utils/index.js

```diff
@@ -0,0 +1,24 @@
+import path from 'path'
+import fs from 'fs'
+import config from '../config'
+import utilsFactory from './factory'
+
+const {
+  isMultiSite,
+  hostFromReq,
+  publicFolder,
+  useStaticsByHost,
+  readHtmlTemplate,
+  hrTimeToMs,
+  buildRequestUrl
+} = utilsFactory({path, fs, config})
+
+export {
```

Comment:
Keep named export in order to keep compatibility in all the places some methods from this file are imported.